src/DigitalTwin/pnp/src/common/utility.ts

```diff
@@ -36,11 +42,15 @@ export class Utility {
    * @param replacement replacement
    * @param caseInsensitive identify if it is case insensitive
    */
-  static replaceAll(str: string, replacement: Map<string, string>, caseInsensitive = false): string {
+  static replaceAll(
+    str: string,
+    replacement: Map<string, string>,
+    caseInsensitive = false
+  ): string {
     const flag = caseInsensitive ? "ig" : "g";
     const keys = Array.from(replacement.keys());
     const pattern = new RegExp(keys.join("|"), flag);
-    return str.replace(pattern, (matched) => {
+    return str.replace(pattern, matched => {
```

Comment:
here is different from tslint of rule "arrow-parens"

Reply:
From https://eslint.org/docs/rules/arrow-parens, it seems no use of parens when they are not required is recommended.